jest.dist.json

```diff
@@ -9,10 +9,15 @@
     "^emotion-utils$": "<rootDir>/packages/emotion-utils/dist/index.cjs.js",
     "^emotion-server$": "<rootDir>/packages/emotion-server/lib",
     "^emotion-theming$": "<rootDir>/packages/emotion-theming/dist/index.cjs.js",
-    "^babel-plugin-emotion": "<rootDir>/packages/babel-plugin-emotion/lib"
+    "^babel-plugin-emotion": "<rootDir>/packages/babel-plugin-emotion/lib",
```

Comment:
@mitchellhamilton would you be interested in using a package that would create those for you? I've created some time ago https://github.com/Andarist/lerna-alias for my use case, but can easily add an option to use `main` field / standard node resolve algo

Reply:
I think we might be able to remove these mappings since it should just work with lerna/yarn linking but I just haven't tried. I looked at lerna-alias before and I tried to use it for the rollup config but it wasn't working, I can't remember exactly why though. I think it would be really good for the main jest config though.